docs/content/en/integrations/parsers/file/wazuh.md

```diff
@@ -2,4 +2,4 @@
 title: "Wazuh Scanner"
 toc_hide: true
 ---
-Import JSON report.
+Import findings from Wazuh. The export from wazuh should be done via the script [available here](https://github.com/quirinziessler/wazuh-findings-exporter). The script fetches the findings by Wazuh client groups and saves them as json, ready for upload. Have in mind to adjust the max file size via "DD_SCAN_FILE_MAX_SIZE" if the file is larger than the default value of 100MB.
```

Comment:
You introduce a new JSON format her which is generated as an additional component between the scanner and DefectDojo. This brings new complexity and the default scanner output can't be processed any more. 

I would suggest that you either include this code into this DefectDojo parser to be not dependant on this additonal script or advance this parser in a way to have two options to upload the output: 

1. Being able to parse the default JSON output directly from Wazuh
2. Using your exporter to generate the new JSON structure and parsing the JSON. 

Furthermore, please extend your testcoverage.

Reply:
@manuel-sommer excellent feedback! 

@quirinziessler The use of this extra script needs to be optional such that the original parser works with our without the preprocessing